.github/CODEOWNERS

```diff
@@ -18,4 +18,7 @@
 /default_app/ @electron/docs
 /docs/ @electron/docs
 /docs-translations/ @electron/i18n
-/npm/ @electron/electrocats
\ No newline at end of file
+/npm/ @electron/electrocats
+
+# WG CODEOWNER Patterns
+/script/ @electron/wg-releases
```

Comment:
this seems overly broad. Can we move all release-related scripts to e.g. `script/release` and protect that instead?

Reply:
my concern is that it'd break all our backports and i'd love to not do that i think

Reply:
i can try though :)